bin/bossSwitch.py

```diff
@@ -0,0 +1,210 @@
+    vaultD = [ASGdescription["AutoScalingGroups"][2]["MinSize"], ASGdescription["AutoScalingGroups"][2]["MaxSize"],ASGdescription["AutoScalingGroups"][2]["DesiredCapacity"]]
+   
+    #Start vault instance
+    print("Starting vault...")
+    client.update_auto_scaling_group(AutoScalingGroupName=vaultg, MinSize = vaultD[0] , MaxSize =  vaultD[1], DesiredCapacity = vaultD[2])
+    client.resume_processes(AutoScalingGroupName=vaultg,ScalingProcesses=['HealthCheck'])
+    time.sleep(120)
+    print("Vault instance running")
+
+    #Import vault content:
+    subprocess.call('./bastion.py ' + args.vpc + ' vault-unseal',shell=True)
+    print("Importing vault content")
+    subprocess.call('./bastion.py ' + args.vpc + ' vault-import < ../config/vault_export.json', shell=True)
+    print(bcolors.WARNING + "Successful import" + bcolors.ENDC)    
+
+    #Start endpoint and activities instances
+    print("Starting endpoint, and activities...")
+    client.update_auto_scaling_group(AutoScalingGroupName=endpoint, MinSize = endpointD[0] , MaxSize = endpointD[1] , DesiredCapacity = endpointD[2])
+    client.resume_processes(AutoScalingGroupName=endpoint,ScalingProcesses=['HealthCheck'])
+    
+    client.update_auto_scaling_group(AutoScalingGroupName=activities, MinSize = activitiesD[0] , MaxSize = activitiesD[1] , DesiredCapacity = activitiesD[2])
+    client.resume_processes(AutoScalingGroupName=activities,ScalingProcesses=['HealthCheck'])
+
+    print(bcolors.OKGREEN + "TheBoss is on" + bcolors.ENDC)
+
+
+def stopInstances():
+    """
+        Method used to stop currently running instances
+    """
+    #Save current ASG descriptions:
+    print("Saving current auto scaling group configuration...")
+    response = client.describe_auto_scaling_groups(AutoScalingGroupNames=[endpoint,activities,vaultg])
+    save_obj(response,'ASGdescriptions')
+    
+    #Export vault content:
+    print("Exporting vault content...")
+    subprocess.call('./bastion.py ' + args.vpc + ' vault-export > ../config/vault_export.json', shell=True)
+    print(bcolors.WARNING + "Successful export" + bcolors.ENDC)
+
+    #Switch off:
+    print("Stopping all Instances...")
+    client.update_auto_scaling_group(AutoScalingGroupName=endpoint, MinSize = 0 , MaxSize = 0 , DesiredCapacity = 0)
+    client.suspend_processes(AutoScalingGroupName=endpoint,ScalingProcesses=['HealthCheck'])
+
+    client.update_auto_scaling_group(AutoScalingGroupName=activities, MinSize = 0 , MaxSize = 0 , DesiredCapacity = 0)
+    client.suspend_processes(AutoScalingGroupName=activities,ScalingProcesses=['HealthCheck'])
+
+    client.update_auto_scaling_group(AutoScalingGroupName=vaultg, MinSize = 0 , MaxSize = 0 , DesiredCapacity = 0)
+    client.suspend_processes(AutoScalingGroupName=vaultg,ScalingProcesses=['HealthCheck'])
+
+    print(bcolors.FAIL + "TheBoss is off" + bcolors.ENDC)
+
+def save_obj(obj, name ):
+    """
```

Comment:
Somewhere it would be good to define the pickled object format, so other developers would know what data it contains.